.changeset/six-rockets-show.md

```diff
@@ -0,0 +1,28 @@
+---
+"wrangler": minor
+---
+
+allow --name and --env args on wrangler deploy
+
+Previously it was not possible to provide a Worker nam as a command line argument at the same time as setting the Wrangler environment.
```

Comment:
My GitHub feed got me here today, probably to make this very important comment:

```suggestion
Previously it was not possible to provide a Worker name as a command line argument at the same time as setting the Wrangler environment.
```

:grin:

Reply:
Awesome @gkalpak! Great working with you again 😄

Reply:
https://github.com/cloudflare/workers-sdk/pull/9299